sbol2/property.py

```diff
@@ -935,7 +935,7 @@ def __init__(self, property_owner, type_uri, reference_type_uri,
         super().__init__(property_owner, type_uri, lower_bound, upper_bound,
                          validation_rules, initial_value)
         self.reference_type_uri = reference_type_uri
-        if self._sbol_owner is not None:
+        if self._sbol_owner is None:
```

Comment:
If this adjusted branch is ever taken, it will break on line 940, because it will attempt to assign to `None.properties`.

@tcmitchell : do you remember why we have this setting property_store to the empty list? It appears to be preventing initialization with init value from being done as expected in at least some cases.

Reply:
I do not.

I think `_sbol_owner` is the pointer to the SBOL object that "owns" this object. It's possible this path was trying to mimic something in the C++ code.

I think if this change passes the new unit test, and this change does not fail any other unit tests, then it should be ok.